src/platform/user/authentication/components/LoginActions.jsx

```diff
@@ -64,4 +73,16 @@ export default function LoginActions({ externalApplication }) {
       </div>
     </div>
   );
-}
+};
+
+LoginActions.propTypes = {
+  showsignInPageAndModalExperiment: PropTypes.bool,
+};
+
+const mapStateToProps = state => ({
+  showsignInPageAndModalExperiment: toggleValues(state)[
+    FEATURE_FLAG_NAMES.signInPageAndModalExperimentLga
+  ],
+});
+
+export default connect(mapStateToProps)(LoginActions);
```

Comment:
Opt for `useSelector` instead of `mapStateToProps`

Reply:
Hey @asg5704 thank you for the feedback. I am pretty new and not that versed with Redux. I did use `mapStateToProps` because of its usage in other parts of the codebase which used it so I followed the pattern of a few other components. I am happy to learn and use - `useSelector` but wondering if it is to be established as a new norm/pattern in the future or the merits of it?

Reply:
`script/github-actions/daily-product-scan/tests/mocks/applications/app-1/HealthCareApp.jsx` for instance here. Also are you talking about just moving the feature flag bit to its own `selectors.js` file?

Reply:
Hey @moanu404

So just some background a lot of the codebase uses old React patterns that while still work could become deprecated in the future as React works towards more hook-based patterns. So the main reason why I recommend using `useSelector` instead of `mapStateToProps` is because `mapStateToProps + connect` passes down multiple values as props with the higher-order component where as `useSelector` looks for one value and does not bind it to props so you get a tiny performance bump.

As for the selector, you can put it in the `platform/user/authentication/selectors.js` file to keep the component relatively clean since there are already the `toggleValues` imports.

If you have any questions, etc. just reach back out.